src/reporters/base-reporter.js

```diff
@@ -87,6 +88,10 @@ export default class BaseReporter {
   peakMemory: number;
   startTime: number;
 
+  logNetworkUsage(bytes: number) {
+    this.networkUsage += parseInt(bytes);
```

Comment:
If it is already defined as `number`, you shouldn't need the extra `parseInt` call, right?

Reply:
Was trying to be 'defensive' here, but maybe that's detrimental to performance. I'll go back and see if there are any cases where `parseInt` would be required.

Reply:
Just trust `flow` ;)